### ISSUE TITLE
Approval Feature for Seller Registration 

### ISSUE THREAD

**Issue by ThanChay** [Arcadier/Mogozay-Support#2]
Current Logic 
Every seller can register by themselves then can create items and sell on Mogozay without approval of company admin side. 

**Expectation Logic**
**1st action**, The seller can register themselves on Mogozay and then Mogozay admin will review their company profile as a accordingly and contact them.

**2nd action**, Mogozay admin will approve. 

**3rd action**, After approval action, send email to seller " Congratulation! you can start selling on Mogozay"

After all above action, Seller can login to their dashboard and can create items.

**Comment by Arcadier** [comment]
I believe what you are looking for is the Private Marketplace option that our admin portal already offers. You can check it out in the Admin Portal by navigating to: `Advanced` -> `Private Settings` -> Turn it on. There's a bunch of options you can play with.

**Comment by ThanChay** [comment]
Can i edit UI for that private maketplace? like the button color change/remove title or something like that

![Screenshot from 2020-01-17 16-21-41](https://user-images.githubusercontent.com/6531873/72602440-add27900-3945-11ea-829f-d3d44ae844d9.png)

**Comment by Arcadier** [comment]
Yes, you can edit the UI, from the same settings page as where you turned on the private setting:

![image](https://user-images.githubusercontent.com/6611854/72708117-fa63c180-3b9c-11ea-886f-02cf6d8a7dda.png)

We can also activate custom code for this page for you if you want.

**Comment by ThanChay** [comment]
so, How can i remove About Us title and background color?

**Comment by Arcadier** [comment]
Yes you can add your custom CSS and JS on to the marketplace via
https://mogozay.arcadier.io/admin/custom-code/index

Do note that you will have to call the specific class names on the page (as the custom css and javascript will be used across the whole marketplace)

**Comment by ThanChay** [comment]
Can we switch on/off this 3 action from admin side? I don't want to allow to register as a buyer role on Arcadier end.

![Screenshot from 2020-01-21 10-45-19](https://user-images.githubusercontent.com/6531873/72775045-4107f900-3c3b-11ea-9820-c5df4f9e093c.png)

**Comment by Arcadier** [comment]
You can use custom code to have the "Register/Sign In" link to redirect to mogozay.com instead of Arcadier's registration page.
